build/ndm-exporter/Dockerfile.in

```diff
@@ -6,5 +19,15 @@
 FROM @BASEIMAGE@
 
 ARG ARCH
+ARG DBUILD_DATE
+ARG DBUILD_REPO_URL="https://github.com/openebs/node-disk-manager"
+ARG DBUILD_SITE_URL="http://www.openebs.io/"
```

Comment:
`DBUILD_REPO_URL` and `DBUILD_SITE_URL`  are passed as arguments. We don't need to set the values again.